.env.example

```diff
@@ -1,50 +1,50 @@
 # Application URL
-VUE_APP_BASE_URL=http://localhost:8080
+VITE_BASE_URL=http://localhost:8080
```

Comment:
Is there a way to keep old names for retro-compatibility with on-premises?

Reply:
I found this on the Internet: https://github.com/ElMassimo/vite-plugin-environment/blob/main/README.md#loading-prefixed-variables